lib/configuration.yaml

```diff
@@ -104,7 +104,7 @@ dev: &dev
       bvAlt: 0
     systemTag: system
 
-  # GodTossing
+  # SSC (formerly known as GodTossing)
   gt: &dev_gt
```

Comment:
I am reviewing commit with message `Rename everything with “gt” or “godtossing”` and I see `gt` :suspect:

Reply:
Why not rename to `ssc`?

Reply:
I didn't rename it because I thought that it would break things. (For the same reason I didn't rename the DB key used for SSC data.) Now that you have mentioned it, I think that it won't break things so I'll rename it.

Reply:
It's better not to rename DB key indeed, but configuration format can be changed without breaking anything, imo.

Reply:
Btw, DB key was and is `ssc`, so there is no need to change it anyway.

Reply:
No, some other DB key.